deployments/development-install/helmfile.yaml

```diff
@@ -15,12 +15,16 @@ releases:
                 - ocis.kube.owncloud.test
 
       - logging:
-          level: debug
+          level: info
 
       - insecure:
           oidcIdpInsecure: true
           ocisHttpApiInsecure: true
 
+      - features:
+          demoUsers: true
+          basicAuthentication: true
```

Comment:
Should we add a note to the Readme, that demoUser and basicAuth are enabled in this deployment?